_source/api/logzio-public-api.yml

```diff
@@ -5601,6 +5808,62 @@ definitions:
         description: Detailed description of the endpoint
         example: Endpoint for sending alerts to Slack
 
+  OpsGenieEndpointUpsertRequest:
+    type: object
+    properties:
+      title:
+        type: string
+        description: Name of the endpoint
+        example: OpsGenie endpoint
+      description:
+        type: string
+        description: Detailed description of the endpoint
+        example: Sends notifications to OpsGenie
+      apiKey:
+        type: string
+        description: API key from OpsGenie
```

Comment:
Please refer to the *Opsgenie* to learn how to set up the API key

https://docs.opsgenie.com/docs/logz-io-integration